src/app/pages/FastBtcPage/components/Deposit/SignatureValidation/index.module.scss

```diff
@@ -0,0 +1,14 @@
+.title {
+  @apply tw-font-semibold tw-tracking-normal tw-mb-8;
+  font-size: 1.375rem;
+  color: #e9eae9;
+  line-height: 1.6875rem;
+}
+.description {
```

Comment:
You don't need to create this selector as you're just using standard Tailwind values + it only has one usage. Use the classes directly in the component.